go.mod

```diff
@@ -366,7 +366,7 @@ require (
 	go.opentelemetry.io/collector/confmap/provider/httpsprovider v0.102.1 // indirect
 	go.opentelemetry.io/collector/confmap/provider/yamlprovider v0.102.1 // indirect
 	go.opentelemetry.io/collector/extension/auth v0.102.1 // indirect
-	go.opentelemetry.io/collector/filter v0.102.0 // indirect
+	go.opentelemetry.io/collector/filter v0.102.1 // indirect
```

Comment:
I don't think this `go.mod` should change, was this just already out of date with `go mod tidy`?

Reply:
yea, i think this was just out of date from the last otel update, somehow